### DISCUSSION TITLE
Quota Limit Work-Around: Multiple Google Cloud Projects

### DISCUSSION THREAD

**Discussion by AaronPhilipp** [ThioJoe/YT-Spammer-Purge#937] [Q&A]
Hi,

I tried your work-around for my project to retrieve data through the API. I had 10 projects and changed the api-key when I ran out of quota. Today I received a mail from google saying:

> Dear Developer,
> We have recently detected that your Google Cloud Project loopsv1 (id: loopsv1) has been circumventing our quota restrictions via multiple projects that act as one and appears to be violating applicable YouTube API Services Terms of Service and Developer Policies.  As a result, we have restricted your use of the corresponding YouTube API Services.

Has anyone experienced the same issue?

Thank you in advance,

Aaron

**Comment by ThioJoe** [comment]
oof, maybe that's a new thing. You can try requesting additional quota: https://support.google.com/youtube/contact/yt_api_form?hl=en

**Reply by AaronPhilipp** [reply]
Yes, I know. But is there anyone who has experience with this? I can't find any testimonials or how many quotas I can apply for.... It would be best if the quotas were unlimited....

**Comment by cochcoder** [comment]
Does their terms of service say that you can't use multiple projects for multiple programs? I just got the same email and I'm requesting an appeal, though any information would be helpful. I'm confused if my current use of the projects actually violate their terms of service; my system is: 15 projects all for different Spammer Purge programs, I use one or more programs at once until my API quota runs out.

**Reply by cochcoder** [reply]
I'm not sure if this is your work-around, but if it isn't and google has some mercy then maybe this would be the best solution for the time being.

**Reply by AaronPhilipp** [reply]
So I think you've chosen the same work-around as I did. From my experience I can say, nothing will happen if you request an appeal. The faster way is to delete the project and wait the 30 days, until it's finally deleted from Google and using keys from another Google Account in the meantime. 

If you use keys from different Google Accounts when you're operating you have a good chance to not get detected. ;)

**Reply by cochcoder** [reply]
Ok, thank you for the info 😁

**Reply by cochcoder** [reply]
You were right as they are making me delete all but one project less than 12 hours after the appeal.

**Reply by AaronPhilipp** [reply]
In case you missed: There exists the possibility to request more quota: https://support.google.com/youtube/contact/yt_api_form?hl=en

But all the experience reports I found so far say the same. Youtube is answering your request very heterogenously... I personally have no experience with this.

**Reply by ilyasyldrm0** [reply]
Any update? :)
